bundles/org.eclipse.jface/src/org/eclipse/jface/viewers/CellEditor.java

```diff
@@ -692,7 +692,7 @@ public boolean isValueValid() {
 	 * @param keyEvent
 	 *            the key event
 	 */
-	protected void keyReleaseOccured(KeyEvent keyEvent) {
+	protected void keyReleaseOccurred(KeyEvent keyEvent) {
```

Comment:
Unfortunately this is API so it can not be changed as every downstream project will break . Please revert this part.

Reply:
If you don't plan to act on this one I'll close the PR as it can't be accepted in this form.